packages/client-utils/test/snapshots/exports.test.js.md

```diff
@@ -0,0 +1,33 @@
+# Snapshot report for `test/exports.test.js`
+
+The actual snapshot is saved in `exports.test.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## index
+
+> Snapshot 1
+
+    [
+      'Natural',
```

Comment:
`Natural` doesn't seem to carry its weight. I don't see any use of it. **suggestion**: How about getting rid of it?